gsk4/sys/src/lib.rs

```diff
@@ -1508,6 +1529,12 @@ extern "C" {
     pub fn gsk_transform_node_get_child(node: *const GskTransformNode) -> *mut GskRenderNode;
     pub fn gsk_transform_node_get_transform(node: *const GskTransformNode) -> *mut GskTransform;
 
+    //=========================================================================
+    // GskVulkanRenderer
+    //=========================================================================
+    pub fn gsk_vulkan_renderer_get_type() -> GType;
```

Comment:
No versions? That's problematic.

Also will the Vulkan renderer exist on all platforms? Will it exist in every GTK build or only if built with Vulkan support?

Reply:
Only if built with vulkan support, it needs a feature gate

Reply:
Yeah, both in sys and non-sys :)

Reply:
good you brought this up as the broadway one wasn't properly configured